openreview/journal/process/recruit_process.py

```diff
@@ -46,6 +48,37 @@ def process(client, note, invitation):
 
 '''.format(ACTION_EDITOR_NAME=ACTION_EDITOR_NAME, SHORT_PHRASE=SHORT_PHRASE)
 
-            return client.post_message(subject, [user], message, parentGroup=ACTION_EDITOR_DECLINED_ID)
+            response =  client.post_message(subject, [user], message, parentGroup=ACTION_EDITOR_DECLINED_ID)
+
+        action = 'accepted' if response == 'Yes' else 'declined'
+
+        if JOURNAL_REQUEST_ID:
+            recruitment_notes = list(openreview.tools.iterget_notes(client, invitation=f'.*/Journal_Request.*/-/Reviewer_Recruitment_by_AE', replyto=JOURNAL_REQUEST_ID))
```

Comment:
using `.*` at the beginning of the regex is not a good idea. Is this because we don't know the openreview support group?

I'm not sure the difficulties of adding the support group as constance here.

Reply:
would using `invitation=(openreview.net|OpenReview.net)/Support/-/Journal_Request.*/-/Reviewer_Recruitment` be better or should I just find a way of adding the support group as a constant here?